pages/linux/qm-importdisk.md

```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Import a VMDK disk image to specific storage:
+
+`qm importdisk {{100}} {{path/to/disk.vmdk}} {{storage_name}}`
+
+- Import a VMDK disk image to specific storage in a specific format:
+
+`qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2 | raw | vmdk}}`
```

Comment:
```suggestion
`qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2|raw|vmdk}}`
```

Mandatory fix: [remove extra spaces around pipe symbol: `--format` values doesn't support `"qcow2 "`, `" raw "`, and `" vmdk"` values](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).